docs/sources/configure-client/language-sdks/java.md

```diff
@@ -116,26 +116,25 @@ for configuration in multiple sources: system properties, environment variables,
-|`PYROSCOPE_EXPORT_COMPRESSION_LEVEL_LABELS`|operates similarly to `PYROSCOPE_EXPORT_COMPRESSION_LEVEL_JFR`, but applies to the dynamic labels part. The default value is set to `BEST_SPEED`.                                                                                                                                                                                                                                           |
-|`PYROSCOPE_ALLOC_LIVE`                   |is a boolean value that enables live object profiling when set to true. It is disabled by default.                                                                                                                                                                                                                                                                                          |
-|`PYROSCOPE_GC_BEFORE_DUMP`               |is a boolean value that executes a `System.gc()` command before dumping the profile when set to true. This option may be useful for live profiling, but is disabled by default.                                                                                                                                                                                                             |
+| Flag                                        | Description                                                                                                                                                                                                                                                                                                                                                                                                 |
+|---------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PYROSCOPE_FORMAT`                          | sets the profiler output format. The default is `collapsed`, but in order to support multiple formats it must be set to `jfr`.                                                                                                                                                                                                                                                                              |
```

Comment:
side note: we should probably switch to jfr by default

Reply:
hehe, I already did it this morning https://github.com/grafana/pyroscope-java/pull/112/files#diff-99f8fe330b16fd778c4e54482fe3f5e9ba425b5dd458fbcf9858e3ef74e0d786R70